src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Database/DatabaseHandler.cs

```diff
@@ -120,6 +120,27 @@ public override Task<InitializeViewResult> InitializeObjectView(InitializeViewRe
                             IsAzureDB = isAzureDB
                         };
 
+                        // Collect the Database properties information
+                        if (!requestParams.IsNewObject)
+                        {
+                            var smoDatabase = dataContainer.SqlDialogSubject as Database;
+                            databaseViewInfo.ObjectInfo = new DatabaseInfo()
+                            {
+                                Name = smoDatabase.Name,
+                                CollationName = smoDatabase.Collation,
+                                DateCreated = smoDatabase.CreateDate.ToString(),
+                                LastDatabaseBackup = smoDatabase.LastBackupDate == DateTime.MinValue ? "None" : smoDatabase.LastBackupDate.ToString(),
```

Comment:
This should be localized

@corivera Did you get a chance to verify whether STS respects the display language setting in ADS?

Reply:
Yeah, the dialog switches to Spanish values when I set ADS' language mode to the same.

Reply:
Although it doesn't look like all fields are returning localized values. I'll look into it

Reply:
Made an issue for that. https://github.com/microsoft/azuredatastudio/issues/23392